relayer/msgs.go

```diff
@@ -156,3 +161,107 @@ func (c *Chain) MsgTransfer(dst *PathEnd, amount sdk.Coin, dstAddr string,
 		timeoutTimestamp,
 	)
 }
+
+// CreateClient creates an sdk.Msg to update the client on src with consensus state from dst
+func (c *Chain) CreateClient(
+	dstHeader *tmclient.Header,
+	trustingPeriod, unbondingPeriod time.Duration,
+	signer sdk.AccAddress) sdk.Msg {
```

Comment:
Thanks for taking this on. Do you think you could remove `signer` as a passed in parameter here and replace its usage with `c.MustGetAddress()` as well as add the inline comments like I did in the other pr?

This is helpful for 2 reasons
1) It helps avoid the bug explained at the top of this file
2) Reduces potential for a bug where you do src.CreateClient(dst.MustGetSigner) - in this case the signer comes from the wrong chain. Not having to check to see if this is a bug also sames a bit of time debugging

Reply:
this should be done for all functions in this file